syntaxes/omg.idl.tmLanguage.json

```diff
@@ -142,7 +142,8 @@
         "type_dcl": {
             "patterns": [
                 {"include": "#constr_type_dcl"},
-                {"include": "#typedef_dcl"}
+                {"include": "#typedef_dcl"},
+                {"include": "#interface_def"}
```

Comment:
We should add a declaration to allow also forward declaration of interfaces.

Reply:
Done.